int32 direction
int32 type
int32 driving_direction
string id
float32 speed_vmax
float32 vmax_dev
int32 lane_type

float64 pointX
float64 pointY
float64 pointHA
